src/main/java/Cars.java

```diff
@@ -0,0 +1,58 @@
+import java.util.*;
+
+public class Cars {
+    private final List<Car> cars;
+
+    Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    List<Car> cars() {
+        return Collections.unmodifiableList(this.cars);
+    }
+
+    void race(Random random, int maxRandomValue) {
+        for (Car car : this.cars) {
+            car.makeMove(random.nextInt(maxRandomValue));
+        }
+    }
+
+    Winners winners() {
+        Location winnerLocation = findMaxLocation();
+        return new Winners(findCarsAtLocation(winnerLocation));
+    }
+
+    private Location findMaxLocation() {
+        Location maxLocation = new Location(0);
```

Comment:
```suggestion
        Location maxLocation = new Location();
```
 Location 생성자에 위치 값을 0으로 기본 생성자를 추가
여러 개의 생성자를 지원하면 할 수록 이 객체를 사용하는 개발자 입장에서는 편해짐을 느껴보면 좋겠다.